app/fixtures/challenges.json

```diff
@@ -0,0 +1,86 @@
+[
+{
+  "model": "app.challenge",
+  "pk": 2,
+  "fields": {
+    "name": "Polar Bear Habitat - Engage users",
+    "description": "The Canadian Polar Bear Habitat is known around the world for its polar bear research and\r\neducation. However, its remote location limits the number of students, teachers, and life-long\r\nlearners that it can reach. In order to connect with more people � and boost its impact the\r\nHabitat would like to be able to deliver a virtual experience. This could include things like\r\nhandouts, interactive activities and graphics, videos, virtual tours, and quizzes as long as\r\nthe user is engaged and can navigate the experience with ease. Here is a dropbox folder with\r\nadditional information and images that you can consult to better understand our challenge\r\nand build your project! Our website is always there for you to visit as well! Good Luck :))",
+    "created": "2020-09-13",
+    "start": "2020-09-13T15:26:34Z",
+    "end": "2020-09-13T15:26:36Z",
+    "image": "challenges/Untitled_drawing_9_fZoMvwT.jpg",
```

Comment:
These images won't work, they all have to just use the default. Remove the image files as well.